spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+
+describe "As a visitor" do
+    # As a visitor
+    before(:each) do
+        beef = Ingredient.create(name: "Ground Beef", cost: 2)
+        salt = Ingredient.create(name: "Salt", cost: 4)
+        tacobell = Ingredient.create(name: "Taco Bell", cost: 9)
+    end
+    
+    it 'Can list all ingredients' do
+        # When I visit '/ingredients'
+        visit '/ingredients'
+        # I see a list of all the ingredients including their name and cost
+        expect(page).to have_content("Ground Beef")
+        expect(page).to have_content("2")
+        expect(page).to have_content("Salt")
+        expect(page).to have_content("4")
+        expect(page).to have_content("Taco Bell")
+        expect(page).to have_content("9")
+    end
+end
```

Comment:
Make sure to start your tests with exactly what you're describing!

```ruby
RSpec.describe "Ingredients Index Page" do
  describe "As a visitor" do
    # ...
  end
end
```

Also, you should aim to make your tests more robust by reducing the amount of ambiguity in your tests. For example, lines 17, 19, 21 just mention that the numbers 2, 4, and 9 need to show up on the page. These tests are too ambiguous, they could show up as a date, phone number, or address and this would mark this as passing even if it's not what you intended it to pass on.

A more robust test would be similar to something like this:

```ruby
expect(page).to have_content("Ground Beef\nCost: 2") # There is no ambiguity in this test since you know exactly where on the page this would be located.
```

Another thing you could do it to use `within` blocks to target a specific spot on the page.